show/vnet.py

```diff
@@ -64,37 +64,90 @@ def advertised_routes(args):
     click.echo(tabulate(table, header))
 
 
+def get_vnet_info(config_db, vnet_name):
+    """
+    Returns a tuple of (vnet_data, interfaces) for a given VNET name.
+    Includes INTERFACE, VLAN_INTERFACE, VLAN_SUB_INTERFACE, PORTCHANNEL_INTERFACE, and LOOPBACK_INTERFACE.
+    """
+    vnet_data = config_db.get_entry('VNET', vnet_name)
+    if not vnet_data:
+        return None, []
+
+    interfaces = []
+    interface_tables = [
+        'INTERFACE',
+        'VLAN_INTERFACE',
+        'VLAN_SUB_INTERFACE',
+        'PORTCHANNEL_INTERFACE',
+        'LOOPBACK_INTERFACE'
+    ]
+
+    for table in interface_tables:
+        intfs_data = config_db.get_table(table)
+        for intf, data in intfs_data.items():
+            if data.get('vnet_name') == vnet_name:
+                interfaces.append(intf)
+
+    return vnet_data, interfaces
+
+
+def format_vnet_output(vnet_name, vnet_data, interfaces):
+    headers = ['vnet name', 'vxlan tunnel', 'vni', 'peer list', 'guid', 'interfaces']
+    row = [
+        vnet_name,
+        vnet_data.get('vxlan_tunnel', 'N/A'),
```

Comment:
do we need the N/A? I think its standard to keep it empty if nothing is present?